data/json/mapgen/fortified scrap yard/fortified scrap yard.json

```diff
@@ -0,0 +1,155 @@
+[
+{
+    "method": "json",
+    "object": {
+      "furniture": {
+        "#": "f_wreckage",
+        "$": "f_null",

```

Comment:
You don't need to define `f_null` furniture at all. Just remove all such definitions.

Reply:
> You don't need to define `f_null` furniture at all. Just remove all such definitions.

That has been removed.